galata/test/jupyterlab/notebook-scroll.test.ts

```diff
@@ -13,12 +14,12 @@ test.use({
     ...galata.DEFAULT_SETTINGS,
     '@jupyterlab/notebook-extension:tracker': {
       ...galata.DEFAULT_SETTINGS['@jupyterlab/notebook-extension:tracker'],
-      windowingMode: 'full'
+      windowingMode: 'none'
```

Comment:
Yup, that's a mistake when copy-pasting when updating this file to maintain the files identical beyond mode change. Thanks!

> Is there a better way to parameterize this set of tests or does the mocking require two nearly copy-pasted files?

There are two options:
- modify settings during on test runtime  - which will extend the runtime for tests and require adding a lot of code which would depend on settings editor layout - so not good
- modify projects with custom settings (preferred) - but this one is a larger modification to our CI because we split jobs by existing projects (jupyterlab/galata/documentation) - so out of scope for this PR

Reply:
There's one other possibly non-trivial difference between these two files when I diff'd them, are you aware of it?